src/ui/src/components/core/content/CoreChatbot.vue

```diff
@@ -299,15 +334,22 @@ import {
 } from "vue";
 import injectionKeys from "@/injectionKeys";
 import CoreChatbotMessage from "./CoreChatBot/CoreChatbotMessage.vue";
-import type { Message } from "./CoreChatBot/CoreChatbotMessage.vue";
+import type {
+	Message,
+	ContentFragment,
+} from "./CoreChatBot/CoreChatbotMessage.vue";
+import { useLogger } from "@/composables/useLogger";
 
 const rootEl = useTemplateRef("rootEl");
 const messageAreaEl = useTemplateRef("messageAreaEl");
 const messagesEl = useTemplateRef("messagesEl");
 const messageIndexLoading: Ref<number | undefined> = ref(undefined);
 const fields = inject(injectionKeys.evaluatedFields);
+const logger = useLogger();
 const files: Ref<File[]> = shallowRef([]);
 const isUploadingFiles = ref(false);
+const pastedImages: Ref<string[]> = shallowRef([]);
+const processingImages: Ref<boolean> = ref(false);
```

Comment:
You can let TS infer the types

```suggestion
const pastedImages = shallowRef<string[]>([]);
const processingImages = ref(false);
```